.pre-commit-config.yaml

```diff
@@ -85,3 +85,22 @@ repos:
             - "--target-version=py36"
             - "--target-version=py37"
             - "--target-version=py38"
+
+  - repo: https://github.com/Riverside-Healthcare/djLint
+    rev: v1.31.1
+    hooks:
+      - id: djlint-reformat
+        args:
+          - OpenOversight/app/templates
+          - --format-js
+          - --format-css
+          - --profile=jinja
+          - --indent=2
+          - --quiet
+      - id: djlint
+        require_serial: true
```

Comment:
Added so that it does not lint every time there are changes made to the template files. It previously was running 3-4x for each `make lint` command.